from launch import LaunchDescription
from launch_ros.actions import Node


def generate_launch_description():
    return LaunchDescription([
        Node(
            package='pose_graph',
            executable='pose_graph',
            name='pose_graph',
            output='screen',
            parameters=[
                {'visualization_shift_x': 0},
                {'visualization_shift_y': 0},
                {'skip_cnt': 0},
                {'skip_dis': 0.0},
                {'pose_graph_save_path': '/home/ri/raw_data/pose_graph/'},
                {'pose_graph_result_path': '/home/tony-ws1/output/'},
            ],
        ),
    ])
